www/i18n/en.json

```diff
@@ -128,13 +128,13 @@
     "to-label": "To Label",
     "show-all": "All Trips",
     "no-trips-found": "No trips found",
-    "choose-mode": "Mode 📝 ",
-    "choose-replaced-mode": "Replaces 📝",
-    "choose-purpose": "Purpose 📝",
-    "choose-survey": "Add Trip Details 📝 ",
-    "select-mode-scroll": "Mode (👇 for more)",
-    "select-replaced-mode-scroll": "Replaces (👇 for more)",
-    "select-purpose-scroll": "Purpose (👇 for more)",
+    "choose-mode": "Mode",
+    "choose-replaced-mode": "Replaces",
+    "choose-purpose": "Purpose",
```

Comment:
Is there some way we can keep the emojis and just silence them to screen readers? I think at least some of these are valuable as a visual aid to sighted users, but I understand that they could make it more confusing for those using screen readers.
It seems like we should be able to keep emojis that are more 'decorative' in nature.

Reply:
As a sighted user, I agree that these emojis help with clarity and look better overall.

My consideration in removing these emojis was less to avoid screen reader "unpronounceable" bugs, and more to avoid the use of non-character characters in a general sense for broader accessibility. Best case scenario, the screen reader skips over these types of characters entirely, but then we might see users with a dyslexic font app having empty boxes [] for these. I did not directly test this, so I think this is a great discussion. I talked more about my thoughts in #3's description:

>

Reply:
I think this is fine for now, but may be a question for Anat and Kunal as they re-start testing the app.